### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by muskaanshah**
Features added: 
Todo list gets stored on localStorage
todo completed functionality added

To add: edit and delete functionality for todos

### CODE REVIEW

**Review by md-amir-gauhar — APPROVED** [review]
Everything looks fine👍

File: src/components/Todo/TodoCheckbox.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+						payload: { value: curTodo.id },
+					})
+				}
+			/>
+			<span
+				className="mx-1 fw-300 fs-0-9"
+				style={{
+					textDecoration: curTodo.todoCompleted && "line-through",
+					textAlign: "left",
+				}}
```
Comment (manismk) [inline src/components/Todo/TodoCheckbox.jsx:20]: Try to avoid using inline style and add it via classname

**Review by samflab — APPROVED** [review]
Looks great, the UI is good. 
Readable code and good folder structure.
One suggestion: change the favicon and the title of the tab.

**Review by manismk — APPROVED** [review]
Looks good. Easy to understand code.

**Pull request merged**